python/vyos/vpp/nat/nat44.py

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2025 VyOS Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from vyos.vpp import VPPControl
+
+
+class Nat44:
+    def __init__(
+        self,
+        interface_in: str,
+        interface_out: str,
+        translation_pool: str,
+    ):
+        self.interface_in = interface_in
+        self.interface_out = interface_out
+        self.translation_pool = translation_pool
+        self.vpp = VPPControl()
+
+    def enable_nat44_ed(self):
+        """Enable NAT44 endpoint dependent plugin
+        Example:
+            from vyos.vpp.nat.nat44 import Nat44
+            nat44 = Nat44()
+            nat44.enable_nat44_ed()
+        https://github.com/FDio/vpp/blob/stable/2410/src/plugins/nat/nat44-ed/nat44_ed.api
+        """
+        self.vpp.api.nat44_ed_plugin_enable_disable(enable=True)
```

Comment:
What's the default value for the number of sessions per thread? The docs use examples like `nat44 plugin enable sessions 10000`. Is the default unlimited?

Reply:
I cant find default value, what I saw some calculations  here https://github.com/FDio/vpp/commit/5f694d1ecfbe2315e9bbcc98a83b83620e6f6b94
```
vpp# show nat44 summary 
max translations per thread: 64512 fib 0
total sessions: 0 (timed out: 0)
tcp sessions:
    total: 0 (timed out: 0)
        established: 0 (timed out: 0)
        transitory: 0 (timed out: 0)
udp sessions:
    total: 0 (timed out: 0)
icmp sessions:
    total: 0 (timed out: 0)
other sessions:
    total: 0 (timed out: 0)
vpp# 

```